src/typescript/azure-openapi-validator/rules/DescriptionTooShort.ts

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { MergeStates, OpenApiTypes, rules } from '../rule';
+import { trimDescription } from './utilities';
+
+export const DescriptionTooShort: string = "DescriptionTooShort";
+const minimumDescriptionLength = 20;
```

Comment:
where does this number come from?
Is this rule accurate enough to be an error?

Reply:
The length was requested to be an error by PMs. Too-short descriptions mean a lack of contextual information.

The specific length in this case was driven by a thought experiment of "what would be the shortest possible description, including abbreviations and acronyms, which would be useful." Production data was not used under the assumption that it would take a long time to comb through.